### ISSUE TITLE
VS2019 extension store can't find MPS 15.0

### ISSUE THREAD

**Issue by digiexchris** [SimpsonGSD/P4EditVS#38]
![image](https://github.com/SimpsonGSD/P4EditVS/assets/5871642/8d5f63e8-84f5-426e-b468-4a13888f0a90)
This dependency doesn't seem to be installable, either through the VIsual Studio Installer or otherwise. I've seen references to it on stack overflow in being an old dependency that's no longer important to include in the visx. Is there a workaround? VS2022 seems to work fine, but I need VS2019 available as well.

**Comment by SimpsonGSD** [comment]
Hi, what version of 2019 are you running?

**Comment by digiexchris** [comment]
Yup, that was the problem, sorry! I updated visual studio, and it installed fine.
